app/controllers/application_controller.rb

```diff
@@ -85,6 +85,7 @@ def authenticate_token
     authenticate_with_http_token do |token, _options|
       @session = Session.find(token)
       return false if @session.nil?
+      Rails.logger.info("Logged in user with id #{@session.uuid}")
```

Comment:
@markolson I'm not sure what you mean by "maybe we could tag part of a session or GA token". I was thinking of the goal here as making it so we could connect all of the logs for a request together and to also connect them to the user so we would have the ability to dig out a series of actions for that user if needed.

Is there a need to log the GA token? 
What do you mean by "part of a session"?